docs/examples/plot_backtracking_sloped_terrain.py

```diff
@@ -0,0 +1,168 @@
+"""
+Backtracking on sloped terrain
+==============================
+
+Modeling backtracking for single-axis tracker arrays on sloped terrain.
+"""
+
+# %%
+# Tracker systems avoid row-to-row shading when the sun is low in the sky
+# by backtracking. The backtracking strategy orients the modules exactly
```

Comment:
```suggestion
# Tracker systems use backtracking to avoid row-to-row shading when the
# sun is low in the sky. The backtracking strategy orients the modules exactly
```